docs/developers_guide/landice/framework.rst

```diff
@@ -3,10 +3,26 @@
 Land-ice Framework
 ==================
 
+ais_observations
+~~~~~~~~~~~~~~~~
+
+The landice framework module ``compass/landice/ais_observations.py`` contains
+observational data for various Antarctic basins.  These are based on the
+ISMIP6 basin definitions, but it need not be limited to those.  The basins do
+not need to be mutually exclusive, so more can be added as needed.
 
 extrapolate
 ~~~~~~~~~~~
 
 The landice framework module ``compass/landice/extrapolate.py`` provides a
 function for extrapolating variables into undefined regions.  It is copied
 from a similar script in MPAS-Tools.
+
+icshelf_melt
```

Comment:
```suggestion
iceshelf_melt
```

Reply:
Addressed in b5c233ae0f709e56b06cc86fd1b3c695854d3c31